app/views/quick_actions/new.html.haml

```diff
@@ -0,0 +1,21 @@
+- if @walkup_available
+	%p= "#{@facility.abbreviation} / #{@instrument.name}"
+
+	%dl
+		%dt Reservation Date
+		%dd= human_date(@reservation.reserve_start_at)
+
+		%dt Reservation Time
+		%dd= human_time(@reservation.reserve_start_at)
+
+	= form_with url: facility_instrument_quick_actions_path(@facility, @instrument) do |f|
+		%h4 Reservation Length
+		- @reservation_intervals.each_with_index do |interval, index|
+			= f.radio_button "reservation_index", index
+			= f.label "reservation_index_#{index}", "#{interval} mins"
```

Comment:
This should work, but I was thinking the input would submit an interval as a value to the controller rather than an index.  What do you think about that?